### DISCUSSION TITLE
Coolify Login Attempts Logs?

### DISCUSSION THREAD

**Discussion by LaurenceJJones** [coollabsio/coolify#2910] [Feature Requests]
Hey All,

I work for @crowdsecurity and we have been asked alot from the community on writing a blog article on protecting coolify instances from security threats. It's easy to integrate the services that coolify spins up, the main question I have is the coolify login portal does not seem to provide any logs on successful or unsuccessful attempts? Am I missing something or these are not logged?

If not this could be a nice addition since we can protect instances from bruteforce (and I know I know you shouldnt expose the login portal to the internet but you'll be surprised how many that will)

**Comment by peaklabs-dev** [comment]
@LaurenceJJones I saw on Discord that you basically finished the integration with coolify?

**Reply by LaurenceJJones** [reply]
> @LaurenceJJones I saw on Discord that you basically finished the integration with coolify?

Didn't see your question here sorry 😞 

Yeah I have 2 guides that are nearly finished:
- CrowdSec installed on Coolify instance
- CrowdSec installed via Coolify template to localhost and remote servers

The discussion topic was to allow us to detect bruteforce attempts against the coolify login panel, as far as I could see there is no log or detection via nginx server since I couldnt find it.

**Reply by peaklabs-dev** [reply]
Nice, can not wait to read the guides. Yes that is correct, we need to have an area where every thing that is done is logged so we have like a full audit log of logins, actions and so on performed in coolify.

**Reply by sparkyfen** [reply]
Have these guides been posted? I'm interested in them.

**Reply by LaurenceJJones** [reply]
> Have these guides been posted? I'm interested in them.

There planned to be released on CrowdSec side in January / at most early February

**Reply by uncaney** [reply]
are there news on this ?

**Reply by LaurenceJJones** [reply]
> are there news on this ?

Unfortunately unforeseen illness knocked me out of action for a month so still recovering and catching up on tasks.

**Reply by gearsandcode** [reply]
Any updates on this @LaurenceJJones ?

**Reply by LaurenceJJones** [reply]
https://www.crowdsec.net/blog/securing-automated-app-deployment-crowdsec-and-coolify
